### PR TITLE
feat: add print_action pass-through command

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: cmd/aspect/printaction/printaction.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+func NewDefaultPrintActionCmd() *cobra.Command {
+	return NewPrintActionCmd(ioutils.DefaultStreams)
+}
+
+func NewPrintActionCmd(streams ioutils.Streams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "print_action",
+		Short: "Prints the command line args for compiling a file.",
+		Long: `Builds the specified targets and prints the extra actions for the given
```
Comment (alexeagle) [inline cmd/aspect/printaction/printaction.go:36]: nit: can these multi-line literals be indented better like by escaping the first newline

```
        `\
First line goes here
more stuff`,
```

**Review by alexeagle — APPROVED** [review]

File: pkg/aspect/printaction/printaction.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ * limitations under the License.
+ */
+
+package printaction
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
```
Comment (cgrindel) [inline pkg/aspect/printaction/printaction.go:23]: nit: Blank line

**Review by cgrindel — APPROVED** [review]

File: pkg/aspect/printaction/printaction.go (hunk 2 above)
Comment (gregmagolan) [reply pkg/aspect/printaction/printaction.go:23]: that one is intentional to separate our internal aspect.build imports and imports from 3rd party deps :)

File: cmd/aspect/printaction/printaction.go (hunk 1 above)
Comment (gregmagolan) [reply cmd/aspect/printaction/printaction.go:36]: they are all like this so could do a follow-up cleanup PR to fix them all up

**Pull request merged**